test/e2e/common/runtime.go

```diff
@@ -44,7 +44,7 @@ var _ = framework.KubeDescribe("Container Runtime", func() {
 				Testname: Container Runtime, Restart Policy, Pod Phases
 				Description: If the restart policy is set to ‘Always’, Pod MUST be restarted when terminated, If restart policy is ‘OnFailure’, Pod MUST be started only if it is terminated with non-zero exit code. If the restart policy is ‘Never’, Pod MUST never be restarted. All these three test cases MUST verify the restart counts accordingly.
 			*/
-			framework.ConformanceIt("should run with the expected status [NodeConformance]", func() {
+			framework.ConformanceIt("should run with the expected status [NodeConformance] [Arktos-CI]", func() {
```

Comment:
May not be essential test case

Reply:
In runtime area it's pretty representative. Better than other runtime cases.

Reply:
This is a test for kubectl. Try not use deprecated resource. It feels like "Kubectl run deployment" would be a better core test.

Reply:
"when running a container with a new image" might be better candidate for core test.

Reply:
"when running a container with a new image" is context, not a case. Under the context, I added one test case. Will show in next iteration.